app/models/preprint.ts

```diff
@@ -41,6 +41,12 @@ export interface PreprintLicenseRecordModel {
     year: string;
 }
 
+const newVersionAllowedReviewsStates = [
+    ReviewsState.ACCEPTED,
+    ReviewsState.WITHDRAWAL_REJECTED,
+    ReviewsState.PENDING_WITHDRAWAL,
```

Comment:
Ah, I didn't know withdrawal can have pending and rejected states. I need to check BE to see if we are handling this properly.

Reply:
Hmmm, interesting... I see what you are saying about some of the reviews_states for pre-mod vs. post-mod lining up to be the same state (ie: for pre-mod, `"reviews_state": "pending"` => `"is_published": false`, but for post-moderation, either `"reviews_state": "pending"` or `"accepted"` => `"is_published": true`... I'm not sure if we can fully ignore the `reviews_state` flag here, as we shouldn't be allowing a new version for "pre-moderation and rejected", right?

Reply:
Yep, as discussed, let's rely on `published_date` instead of `is_published`. There will be a BE fix to make sure `is_latest_version` checks `published_date` instead of `is_published`.